### ISSUE TITLE
CLIP Loss in ClipLoss Module

### ISSUE THREAD

**Issue by FunkyFrog1** [HaitaoWuTJU/Uncertainty-aware-Blur-Prior#5]
In the implementation, the loss function in `base.utils.ClipLoss` appears to be a standard CLIP loss. However, the paper specifically mentions using the SCE (Symmetric Cross-Entropy) loss function. Could you please clarify this discrepancy?

我注意到在代码实现中，`base.utils.ClipLoss` 的损失函数是标准的 CLIP 损失，但论文中明确提到使用的是 SCE（对称交叉熵）损失函数。请问是不是哪里搞错了
